js/core/utils/drop_down.js

```diff
@@ -0,0 +1,20 @@
+import windowUtils from './window';
```

Comment:
let's move this file to `js/ui/drop_down_editor`

Reply:
fixed